go/runtime/registry/config.go

```diff
@@ -103,6 +103,17 @@ const (
 	RuntimeModeClientStateless RuntimeMode = "client-stateless"
 )
 
+const (
+	// RuntimeEnvironmentSGX specifies to run the runtime in SGX.
+	RuntimeEnvironmentSGX = "sgx"
+	// RuntimeEnvironmentELF specifies to run the runtime in the OS address space.
+	//
+	// Use of this runtime environment is only allowed if DebugDontBlameOasis flag is set.
+	RuntimeEnvironmentELF = "elf"
+	// RuntimeEnvironmentELF specifies to run the runtime in the most appropriate location.
```

Comment:
```suggestion
	// RuntimeEnvironmentAuto specifies to run the runtime in the most appropriate location.
```

Reply:
Oh, weird. I thought gofumpt catches this?

Reply:
Hm indeed, it usually does :thinking: